near/omni-bridge/src/lib.rs

```diff
@@ -458,13 +458,6 @@ impl Contract {
         amount: U128,
         init_transfer_msg: InitTransferMsg,
     ) -> PromiseOrPromiseIndexOrValue<U128> {
-        // Avoid extra storage read by verifying native fee before checking the role
-        if init_transfer_msg.native_token_fee.0 > 0
-            && self.acl_has_role(Role::NativeFeeRestricted.into(), signer_id.clone())
```

Comment:
Maybe we should remove this role and make it so that the implicit account always pays for the native_fee. What functionality would break in that case?